packages/ui-components/src/components/text-field/text-field.tsx

```diff
@@ -0,0 +1,70 @@
+import { Component, Event, EventEmitter, h, Prop } from '@stencil/core';
+import { isEmpty } from 'lodash-es';
+import { EInputFieldType, IErrorMsgs } from './text-field.types';
+
+@Component({
+    tag: 'kv-text-field',
+    styleUrl: 'text-field.scss',
+    shadow: true,
+})
+export class TextField {
+    @Prop({ reflect: true }) type!: EInputFieldType;
+
+    @Prop({ reflect: true }) label: string;
+	@Prop({ reflect: true }) inputName: string;
```

Comment:
fix the indentation and add the correct comment (if you run the lint you have a warning here)

Reply:
@ricardogoncalves89 About the indentation, I verified here and is everything ok, it seems same visualization problem here in GitHub, if you try open the visualization in this link https://github.com/kelvininc/ui-components/blob/text-field/packages/ui-components/src/components/text-field/text-field.tsx is possible to see the correct indentation.

Reply:
@ricardoarasilva check the 'raw' view, there definitely is something wrong with the indentation, do you have whitespace visible in your code editor? Seems like you're using spaces for some lines and tabs for others.

Reply:
@pedrorebelocosta issue found and fixed